### DISCUSSION TITLE
wire-elements Support

### DISCUSSION THREAD

**Discussion by jcabanillas** [nWidart/laravel-modules#1968] [Q&A]
Hello;
I'm using this package [wire-elements](https://github.com/wire-elements/modal)

When I do this:
`
<button type="button" wire:click="$dispatch('openModal', { component: 'ventas::components.venta-detalle-modal' })" class="btn btn-success"> <i class="bi bi-plus"></i> Producto </button>
`
This error appears :

**Livewire\Exceptions\ComponentNotFoundException
Unable to find component: ventas::components.venta-detalle-modal**

My module is under path Modules/Ventas/app/Livewire/Components/VentaDetalleModal.php

Any help please
Thanks.

**Comment by dcblogdev** [comment]
Have you installed the Livewire package?

https://github.com/mhmiton/laravel-modules-livewire

if not you will need to manually register the components.
